src/biokbase/narrative/tests/test_jobmanager.py

```diff
@@ -318,14 +304,10 @@ def test_cancel_jobs__check_job_terminated_returns_true(self, mock_cancel_job):
     @mock.patch("biokbase.narrative.jobs.jobmanager.clients.get", get_mock_client)
     def test_cancel_job__run_ee2_cancel_job(self):
         """cancel a job that runs cancel_job on ee2"""
-        new_job = phony_job()
```

Comment:
no need to create this phony job -- use one of the existing jobs registered when the job manager starts up